src/components/DataArchives/DataArchives.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  root: {
+    backgroundColor: 'white'
+  },
+  main: { paddingBottom: '3rem' },
+  link: { color: theme.palette.primary.dark },
+  typography: {
+    paddingTop: theme.spacing.unit * 6,
+    textAlign: 'center',
+    fontWeight: 'bold'
+  },
+  title: {
+    fontWeight: 700,
+    fontSize: '1rem'
+  },
+  archive: { paddingTop: '2rem', paddingBottom: '2rem' }
+});
+
+const now = 'http://api.airquality.codeforafrica.org/v1/now/';
+const data = 'http://api.airquality.codeforafrica.org/v1/data/';
+const sensors = 'http://api.airquality.codeforafrica.org/v1/sensor/{apiID}/';
+const query = 'http://api.airquality.codeforafrica.org/v1/filter/{query}';
+//const wiki: = 'https://github.com/CodeForAfricaLabs/sensors.AFRICA-AQ-sensors-software/wiki/APIs#api-httpapiairqualitycodeforafricaorg';
+const type = '{sensor type}';
+const area = '{lat, lon, distance}';
+const box = '{lat1, lon1, lat2, lon2}';
+const countryCode = '{country code}';
+
+function DataArchives({ classes }) {
+  return (
+    <Grid
+      container
+      justify="center"
+      alignItems="center"
+      className={classes.root}
+    >
+      <Grid item xs={6} justify="center" align="left" className={classes.main}>
```

Comment:
Why are we using half screen only `xs={6}`? Will a small screen also be able to show all the content in half screen?